flexget/plugins/input/anilist.py

```diff
@@ -122,54 +122,65 @@ def on_task_input(self, task, config):
             try:
                 list_response = list_response.json()['data']
                 logger.debug('JSON output: {}', list_response)
-                for list_status in list_response['collection']['statuses']:
+                for list_status in list_response.get('collection', {}).get('statuses'):
```

Comment:
while you're going overly defensive, maybe add a default to `.get('statuses')`

Reply:
If it's not defined \ empty, won't the loop just be gracefully skipped as intended?

Reply:
no, if it doesn't exist the value will be `None` and it'll crash when you try to iterate over it.
try this:
```suggestion
                for list_status in list_response.get('collection', {}).get('statuses', []):
```

Reply:
It is highly unlikely that field would be empty since the status lists are a default structure and AFAIK can't be deleted by the user, but one can never be too safe. Implemented 👍